config/development.toml

```diff
@@ -262,3 +262,6 @@ refund_duration = 1000
 refund_tolerance = 100
 refund_retrieve_duration = 500
 refund_retrieve_tolerance = 100
+
+[delayed_session_response]
+connectors_with_delayed_session_response = "trustpay"
```

Comment:
needs to be updated in other files as well?
`config.example.toml` & `docker_compose.toml`